policy/scan/local_scanner.go

```diff
@@ -443,7 +451,9 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 			})
 
 			// shut down all ephemeral runtimes
-			runtime.Close()
+			if asset.Connections[0].Type != "k8s" {
```

Comment:
I'm, not sure we will remember why this is there, when we look at it later.

Perhaps we better expose https://github.com/mondoohq/cnquery/pull/2050/files#diff-7263c3839c73627bf39872d6eafc82a803d9b052ece899a5a14fb5cf8497d844R40 and check for it here?

My current gut feeling is, that we will re-use this also for container registries, AWS, ...

Reply:
if seems like this condition isn't needed at all since the shutdown thing already checks that only ephemeral providers are killed